src/gridcoinresearchd.cpp

```diff
@@ -78,6 +71,23 @@ bool AppInit(int argc, char* argv[])
             return false;
         }
 
+        if (mapArgs.count("-version") || mapArgs.count("--version"))
```

Comment:
only -version needed. we interpret --foo and -foo as the same when pushing args to the map

Reply:
Even better, thanks 🙂 

I updated `-help` also to match.